### ISSUE TITLE
Developer Tools pop up took the entire screen

### ISSUE THREAD

**Issue by TinyMito** [responsively-org/responsively-app#1118]
# 🐞 bug report

### ✍️ Description

When the DevTools open, it goes full screen and almost cut off the control on the right side. I'm on 4K monitor.

### 🕵🏼‍♂️ Is this a regression?

New User

### 🔬 Minimal Reproduction

1. Open any website.
2. Open DevTools

### 🌍 Your Environment

<pre><code>Latest version of the app and on macOS Sonoma 14 on MacBook Pro 14 M1 Pro
</code></pre>

### 🔥 Exception or Error or Screenshot

<pre><code>https://github.com/responsively-org/responsively-app/assets/75095713/8f5c5234-19dc-4684-a478-b1a74070d5f9
</code></pre>

**Comment by manojVivek** [comment]
Yeah, this is a strange bug that happens sporadically during the first launch, and we're yet to discover the root cause for this. 

### Workaround
As you have shown in the video, docking the window to the right and then to the bottom would fix it. 
Or if you resize the window a bit, then it would align right back.
